app/config/translations/text_de.yml

```diff
@@ -56,40 +56,40 @@ text:
     text: Mit der rokka image API verteilen Sie Ihre Bilder ganz einfach auf Ihre Web-Präsenzen. Noch einfacher wird die Bedienung von rokka durch unsere Libraries für die gängigen Sprachen.
 
   pricing:
-    title: 'Preise: Einfach und transparent.'
+    title: 'Preise: Fair und transparent.'
     paid:
-      title: Ein Angebot für alle.
-      text: Bei rokka bezahlen Sie nur das, was Sie effektiv brauchen. Die Kosten werden Ihnen quartalsweise in Rechnung gestellt und anhand der monatlichen Aufstellung haben Sie eine genaue Übersicht über Ihre Kosten.
+      title: Professionell einsteigen
+      text: Bei rokka bezahlen Sie nur das, was Sie effektiv brauchen. Wir unterscheiden dabei in Speicherplatz und Traffic. Die Kosten werden Ihnen quartalsweise in Rechnung gestellt.
       monthly: pro monat
       storage_text1: pro 1 GB Speicherplatz
       storage_text2: bis 400 GB
-      traffic_text1: pro 1 GB Datenvolumen
+      traffic_text1: pro 1 GB Traffic
       traffic_text2: bis 2000 GB pro Monat
     details:
-      title: Ab 400 GB Speicherplatz oder 2000 GB Datenvolumen zahlen Sie weniger.
-      text: Mehr erfahren
+      title: Sie wollen noch mehr? Je mehr Sie verbrauchen, desto weniger zahlen Sie. Profitieren Sie von unseren
+      text: günstigen Tarifen.
       storage: Speicherplatz
-      traffic: Datenvolumen
+      traffic: Traffic
       amount: Menge (GB)
       amount_monthly: Menge (GB pro Monat)
       chf_per_gb_monthly: CHF/Monat pro GB
       chf_per_gb: CHF pro GB
       contact: Kontaktieren Sie uns für eine individuelle Offerte.
       owncdn:  Haben Sie Ihr eigenes CDN?
     free:
-      title: Kostenlose Version
-      text: Sie möchten die Katze nicht im Sack kaufen? Das verstehen wir und deshalb können sie alle Funktionen von rokka bis zu 2GB Speicherplatz und 5GB Datenvolumen kostenlos nutzen.
+      title: Kostenlos testen
```

Comment:
this is not the same thing. afaik we offer a free version of rokka with some limits for traffic and storage. of course people can use the free version to try it out, but its also ok to keep using that version for free when people have a small site.

Reply:
we discussed this in the rokka team and we want to keep saying that this is a free version and not just a trial.  can we keep "Kostenlose Version"?